pkg/state/dummy_provider.go

```diff
@@ -0,0 +1,58 @@
+/*
+2023 NVIDIA CORPORATION & AFFILIATES
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package state
+
+import (
+	"github.com/Mellanox/network-operator/pkg/clustertype"
+	"github.com/Mellanox/network-operator/pkg/nodeinfo"
+	"github.com/Mellanox/network-operator/pkg/staticconfig"
+)
+
+type dummyProvider struct {
+}
+
+func (d *dummyProvider) GetClusterType() clustertype.Type {
+	return clustertype.Kubernetes
+}
+
+func (d *dummyProvider) IsKubernetes() bool {
+	return true
+}
+
+func (d *dummyProvider) IsOpenshift() bool {
```

Comment:
this may be problematic if a state renders different containers in case of openshift ?

e.g DTK

Reply:
ah, this is just to get current UT working right ? if so then disregard this comment.